west.yml

```diff
@@ -88,7 +90,8 @@ manifest:
       revision: 31acbaa36e9e74ab88ac81e3d21e7f1d00a71136
       path: modules/lib/gui/lvgl
     - name: mbedtls
-      revision: 53044168b43f06ec5654b906a2cdd260bf477edd
+      remote: d3zd3z
```

Comment:
you should be pointing to the PR in the mbedtls module , not some private remote:

`revision: pull/23/head`